### ISSUE TITLE
PassAndroid opens PDF from browser directly

### ISSUE THREAD

**Issue by spaetz** [ligi/PassAndroid#474]
PassAndroid 3.7.3 (but this has been an issue for a long time). When opening a pdf file from Firefox, it is directly imported and opened by PassAndroid without ever asking.
I have multiple other  applications which can handle PDFs.

When saving the PDF and opening it from the file manager, I do get asked which application to use.

So I guess PassAndroid takes over the URL handler for pdf files as the sole handler? If there is anything I can do to debug things, let me know.

Thanks for PassAndroid, I am enjoying it very much!

**Comment by rivaldi8** [comment]
This also happens to me, but it seems it's Firefox's fault. If I try with Chrome it asks me to choose an application. Could you try it?

**Comment by spaetz** [comment]
Yeah, Chrome opens pdfs in a different app (it does not ask me either). That is weird, but apparently not Passandroid's fault then... Feel free to close unless someone finds a solution.

**Comment by ligi** [comment]
maybe open a issue in the firefox app then? Guess they are not using the intent system in this case.

**Comment by Crashillo** [comment]
In my device I notice that PassAndroid, apart from other hosts as _mobile.aegeanair.com_, _m.aircanada.ca_, etc... has also another compatible device: "*". Looking at the manifest I see this:
https://github.com/ligi/PassAndroid/blob/ce1c46b8aad8011940f626b67b250c690a6b8e22/android/src/main/AndroidManifest.xml#L560-L605

So I believe Android understands that all pdf files, no matter which host they come from, should be open automatically using PassAndroid

**Comment by Scumpeter** [comment]
Someone open-ended a bug for Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1839794

**Comment by spaetz** [comment]
Current Febnec opens pdfs directly withon Firefox and offers a 'Diwnload' and 'Ooen in App' button. Both work as expected, so the issue has been solved for me. Should I close this one?
